java/bundles/org.eclipse.set.feature.table/OSGI-INF/l10n/bundle.properties

```
@@ -22,3 +22,15 @@ TableErrorTableColumns_Source=Quelle
 TableErrorTableColumns_TableType=Zustand
 TableErrorTableColumns_Message=technischer Fehlertext
 TableErrorTableColumns_RowNumber=Zeilennummer
+
+TableExportPart_TaskMsg=Tabellenexport
+
+TableExportPart_ExportPlanningRecordsDescriptionService_ViewName=Dokumentensatz erstellen
+TableExportPart_ExportInventoryDescriptionService_ViewName=Bestandsdokumente erstellen
+TableExportPart_Export_button=Dokumentensatz erstellen
+TableExportPart_ExportInventoryRecords=Bestandsdokumente erstellen
+TableExportPart_TaskMsg=Tabellenexport
+TableExportPart_ExportInventoryDescription=Ausgabe des Zielzustands der Planung.
+TableExportPart_ExportPlanningDescription=Ausgabe des rot/gelb Vergleichs (pdf) und des Zielzustands der Planung (xlsx).
+TableExportPart_ExportStateDescription=Ausgabe des Einzelzustands der PlanPro-Datei.
+TableExportPart_FilterEmptyButton=leere Tabelle abwählen
```

@TruongQuangSB Just one more correction. I was not seeing it before properly. I am sorry

```suggestion
TableExportPart_FilterEmptyButton=leere Tabellen abwählen
```